src/components/volunteer/VolunteerFormCalendar.tsx

```diff
@@ -389,6 +390,14 @@ export const VolunteerFormCalendar = ({
           title={texts.volunteer.save}
           disabled={isLoading}
         />
+        {!!calendarData && (
+          <Button
+            disabled={isLoading}
+            invert
+            onPress={() => deleteCalendarAlert(onCalendarDelete)}
+            title={texts.volunteer.delete}
+          />
+        )}
```

Comment:
i will make the delete button more distinguishable

<img width="378" alt="image" src="https://user-images.githubusercontent.com/1942953/199058383-ad4f7033-526c-4500-891d-f683f5cd4400.png">

Reply:
01b627a